providers/implementations/signature/rsa_sig.c

```diff
@@ -1017,8 +1432,9 @@ static void rsa_freectx(void *vprsactx)
     if (prsactx == NULL)
         return;
 
-    EVP_MD_CTX_free(prsactx->mdctx);
-    EVP_MD_free(prsactx->md);
+    wrap_md_freectx(prsactx);
+    OPENSSL_free(prsactx->sig);
+    EVP_MD_free(prsactx->md.md);
```

Comment:
This might be a dumb question, but is it ok to free the mdctx before freeing the associated md here?

Reply:
Yes it is!  That's even the right way to do it